src/components/datatables/Actions.tsx

```diff
@@ -197,14 +197,7 @@ export function Actions(props: Props) {
           props.defaultOptions &&
           !props.withoutStatusFilter && (
             <Select
-              styles={{
```

Comment:
@beganovich The "customStyles" object by itself includes the same styling for the multiValue prop, but excluding the customStyles object from styling the whole select dropdown is breaking dark mode adjustments. Please let me know if there is some edge case that this styling object does not cover.